src/main/java/de/tum/in/www1/artemis/config/migration/DatabaseMigration.java

```diff
@@ -171,9 +171,8 @@ private String getPreviousVersionElseThrow() {
             if (result.next()) {
                 return result.getString("latest_version");
             }
-            // if no version exists, we fail here
-            log.error(error);
-            System.exit(12);
+            // no version exists
```

Comment:
```suggestion
            // no version exists
            // we can proceed by creating the initial schema
```

Reply:
Actually, it is more likely that the initial schema was already created, but Artemis failed to start completely due to other faulty config. This then leaves the `artemis_version` table empty.

However, I can adjust the comment to say something like "no version exists, so we continue"